www/md_base/src/styles/styles.less

```diff
@@ -4,3 +4,7 @@
 [ng-cloak] {
   display: none !important;
 }
+
+a {
+  text-decoration: none;
```

Comment:
We need to differenciate clickable from unclickable.
![image](https://cloud.githubusercontent.com/assets/109859/7348692/e4c1579a-ecf3-11e4-8a40-0efe5892e081.png)

here, "API Reference" is not clickable, "/application.spec" is clickable, but are in the same color, its hard to make a difference.